domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/FullDefenceResponse.java

```diff
@@ -70,8 +68,8 @@ public DefenceType getDefenceType() {
         return defenceType;
     }
 
-    public String getDefence() {
-        return defence;
+    public Optional<String> getDefence() {
```

Comment:
didn't get why optional when minimum size is 1

Reply:
Minimal size is 1 when defence is submitted to prevent accepting `""`. Optional is needed because I removed constraint that required the value so now submitting `null` or `undefined` is possible.